sixtyfps_runtime/rendering_backends/gl/lib.rs

```diff
@@ -934,7 +934,7 @@ impl ItemRenderer for GLItemRenderer {
             return;
         }
 
-        let cache_entry = self
+        let cache_entry = match self
```

Comment:
style: this expression is a bit big and the match arm are far away from the match.  Maybe consider using an intermediate variable.

Reply:
That's a very good point - thanks. I'll change that with an intermediate variable.